spec/features/confirmation_page_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+feature 'Confirmation page' do
+  include FormMethods
+
+  before do
+    visit returning_user_session_path
+    fill_in_return_form claim.reference, 'lollolol'
+
+    visit claim_review_path
+    click_button 'Submit'
+  end
+
+  around do |example|
+    travel_to(Time.new 2014) { example.run }
```

Comment:
Don't have to reset it. I'm assuming this helper behaves like Timecop and needs resetting?